app/assets/stylesheets/base/_tables.scss

```diff
@@ -41,21 +42,16 @@ th {
   vertical-align: middle;
 }
 
+th.actions {
+  width: 300px;
+}
+
 caption {
   margin: $small-spacing 0;
   text-align: left;
   width: 75%;
 }
 
-.table-section-header {
```

Comment:
@derekprior This is actually being used for the activity feed now, just FYI for when you merge because it's being removed here and then used in another branch.